### ISSUE TITLE
Parser error on whitespace after the root element (allowed as per NT-document)

### ISSUE THREAD

**Issue by SyKingW** [horazont/rxml#1]
```rust
let content = r#"<?xml version="1.0"?>
            <!--
            code comment
            -->
            <configurator>
            <include client="false" server="true" clientLocked="true" serverLocked="true">Basic</include>
            </configurator>
                        "#.to_string();
```

not-well-formed: invalid syntax: malformed cdata section start

```rust
let content = r#"<?xml version="1.0"?>
<configurator>
<include client="false" server="true" clientLocked="true" serverLocked="true">Basic</include>
</configurator>
"#.to_string();
```

not-well-formed: unexpected Text token at end of document (expected end-of-file)

**Comment by horazont** [comment]
Hi and thanks for the report.

> ```rust
> let content = r#"<?xml version="1.0"?>
>             
>             <configurator>
>             <include client="false" server="true" clientLocked="true" serverLocked="true">Basic</include>
>             </configurator>
>                         "#.to_string();
> ```
> 
> not-well-formed: invalid syntax: malformed cdata section start

This is expected. Comments are deliberately not supported and they look like CDATA sections to the parser, hence the (slightly odd) error message.

> ```rust
> let content = r#"<?xml version="1.0"?>
> <configurator>
> <include client="false" server="true" clientLocked="true" serverLocked="true">Basic</include>
> </configurator>
> "#.to_string();
> ```
> 
> not-well-formed: unexpected Text token at end of document (expected end-of-file)

This is indeed a bug because whitespace after the last element is allowed as per the standard, so I am renaming this issue to narrow down the scope.

**Comment by horazont** [comment]
@SyKingW Fixed in version 0.3.0
